commands/new_function.go

```diff
@@ -90,20 +89,24 @@ func runNewFunction(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("%s is unavailable or not supported", language)
 	}
 
-	appendMode := len(appendFile) > 0
-	if appendMode {
-		if (strings.HasSuffix(appendFile, ".yml") || strings.HasSuffix(appendFile, ".yaml")) == false {
-			return fmt.Errorf("when appending to a stack the suffix should be .yml or .yaml")
+	appendMode := false
+	fileProvided := len(yamlFile) > 0
```

Comment:
This can evaluate to true if `--yaml` is absent but `stack.yml` exists.  Is that the intended behaviour?